psydac/feec/global_projectors.py

```diff
@@ -275,11 +275,11 @@ def __init__(self, L2, quads=None):
         slices     = tuple(slice(p+1,-p-1) for p in L2.degree)
```

Comment:
L2 projector has a 'quads' argument while Hcurl and Hdiv projs have 'n_quads'  arguments. Why ?

Reply:
This was an error, I fixed it in commit 3367105